client/src/components/pages/application/Availability/index.js

```diff
@@ -1,5 +1,117 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useHistory } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+import PropTypes from 'prop-types';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import NonLinearStepper from '../../../application/NonlinearStepper';
+import Alert from '../../../common/Alert';
+import Typography from '../../../common/Typography';
+import Button from '../../../common/Button';
+import Limitation from '../../../common/limitation';
+import backGround from '../../../../assets/images/backGround.svg';
+import { availability } from './availability.json';
+import './index.css';
 
-const Availability = () => <h1>Availability</h1>;
+const Availability = ({ fullName, userID }) => {
+  const [available, setAvailable] = useState(false);
+  const [alert, throwAlert] = useState(false);
+  const [nonLinear, setNonLinear] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const history = useHistory();
+
+  const next = async () => {
+    if (!available) throwAlert(true);
+    else {
+      setLoading(true);
+      try {
+        await axios.patch(`/api/v1/applicants/${userID}`, {
+          available,
+        });
+        setLoading(false);
+        setNonLinear(true);
+      } catch (error) {
+        throwAlert(true);
+      }
+    }
+  };
+
+  useEffect(() => {});
```

Comment:
get User ID inside useEffect to get user data , also if you can change setAvailable value through it 
```suggestion
useEffect(() => {
getUserID().then((data) => {
      if (data.message === 'you are authorized') {
        setId(data.userId);
      }
    });
    }, [UserId]);
```